packages/gspell.rb

```diff
@@ -0,0 +1,30 @@
+require 'package'
+
+class Gspell < Package
+  description 'a flexible API to implement the spell checking in a GTK+ application'
+  version '1.9.1'
+  compatibility 'all'
+  source_url 'https://download.gnome.org/sources/gspell/1.9/gspell-1.9.1.tar.xz'
+  source_sha256 'dcbb769dfdde8e3c0a8ed3102ce7e661abbf7ddf85df08b29915e92cd723abdd'
+
+  depends_on 'gtk3'
+  depends_on 'enchant'
+  depends_on 'libxml2'
+  depends_on 'iso_codes'
+  depends_on 'gobject_introspection' => ':build'
+  depends_on 'vala' => ':build'
+  depends_on 'gtk_doc' => ':build'
+
+  ENV['CC'] = "clang"
+  ENV['CXX'] = "clang"
+  ENV['XML_CATALOG_FILES'] = "#{CREW_PREFIX}/etc/xml/catalog"
+
+  def self.build
+    system "./configure --help"
```

Comment:
Why do we need this line?

Reply:
"Also enabled a ./configure --help line because just like with meson configure build if you're building from the source it is useful to know what configure options are available."

I was just looking for an equivalent to ```meson configure build``` to show what the options are.

I can delete it if is irritating.

Reply:
I just thought it was something extra that you added to help with the options but forgot to remove.

Reply:
Nope. I'm realizing when modifying packages that seeing what the options are is actually really helpful, without having to do a ```crew install -k``` and going into the install directory. Maybe I should start breaking out a pre-build section?